postman/collection-dir/adyen_uk/Flow Testcases/QuickStart/Payouts - Create/event.test.js

```diff
@@ -0,0 +1,60 @@
+// Validate status 2xx
+pm.test("[POST]::/payouts/create - Status code is 2xx", function () {
+  pm.response.to.be.success;
+});
+
+// Validate if response header has matching content-type
+pm.test(
+  "[POST]::/payouts/create - Content-Type is application/json",
+  function () {
+    pm.expect(pm.response.headers.get("Content-Type")).to.include(
+      "application/json",
+    );
+  },
+);
+
+// Validate if response has JSON Body
+pm.test("[POST]::/payouts/create - Response has JSON Body", function () {
+  pm.response.to.have.jsonBody();
+});
+
+// Validate if status is successful
+// if (jsonData?.status) {
+//   pm.test("[POST]::/payouts/create - Content check if value for 'status' matches 'success'",
+//     function () {
+//       pm.expect(jsonData.status).to.eql("success");
+//     },
+//   );
+// }
```

Comment:
Marking this, which needs to be uncommented once the things are sorted out.  
CC: @bernard-eugine